internal/diff/result_test.go

```diff
@@ -0,0 +1,63 @@
+package diff
+
+import (
+	"bytes"
+	"github.com/Zxilly/go-size-analyzer/internal/printer"
+	"github.com/Zxilly/go-size-analyzer/internal/test"
+	"github.com/go-json-experiment/json"
```

Comment:
Do you need really need this experimental lib?

Maybe you could add a comment about why you need it

Reply:
This is for streaming json encoding, where the memory footprint may balloon unacceptably when analyzing certain megafiles.